### ISSUE TITLE
Could you please publish some pre-trained model weights for some other datasets like ImageNet? 

### ISSUE THREAD

**Issue by cold-blue** [JiahuiYu/generative_inpainting#344]
Could you please publish some pre-trained model weights for some other datasets like ImageNet? For some reason, I need a pre-trained model which is trained on ImageNet. I don’t have conditions to train the model but still want to use it to Inpaint my masked images. Thanks a lot.

**Comment by JiahuiYu** [comment]
We have a pretrained model on ImageNet for DeepFill v1. I didn’t train one for DeepFill v2.
